packages/language/src/preprocessor/instruction-interpreter.ts

```diff
@@ -857,7 +859,8 @@ function runInclude(item: IncludeItem, context: InterpreterContext): void {
     const instruction = generateInstructions(subProgram.statements);
     const newContext: InterpreterContext = {
       ...context,
-      uri: uri,
+      currentUri: uri,
+      entryUri: context.entryUri,
```

Comment:
Minor: We don't need to explicitly state this, since we use the `...context` spread operation, which already sets `entryUri` to `context.entryUri`.